data-service/README.md

```diff
@@ -41,3 +41,76 @@ To run Prettier
 ```
 npm run pretty-quick
 ```
+
+## Build Docker container
+
+```
+docker build -t core-banking/data-service .
+```
+
```

Comment:
Can you also please add details about ACCOUNTS_HOST and TRANSACTIONS_HOST env variables in readme?

Reply:
+ PORT as well pls :) and any other env var needed to configure the service.

Reply:
added in the new PR
https://github.com/DeloitteDigitalAT/core-banking/pull/25